draft-ietf-oauth-device-flow.xml

```diff
@@ -259,12 +257,14 @@
 	<t>
 	  For example, the client makes the following HTTPS request (line
 	  breaks are for display purposes only):
+  </t>
+  <t>
 	  <figure><artwork><![CDATA[
-   POST /token HTTP/1.1
+   POST /device_authorization HTTP/1.1
```

Comment:
The important point is that endpoints used in the "Device Authorization Request" and the "Device Access Token Request" are not the same endpoints. It was a bug in `-04` that the same name was used in both examples, which could give a misleading impression that they were the same endpoint.

I like the term "device_authorization" since it matches the terms we're using in the spec for that action, but if you want to make this proposal on the oauth IETF list to change it to "associate", we can do a poll and see what others think.